### DISCUSSION TITLE
Decoupling particle radius and specific surface area

### DISCUSSION THREAD

**Discussion by fussegli** [pybamm-team/PyBaMM#2254] [Q&A]
Hi everyone,
I try to make a comparison between an in-house microstructure-scale 3D model (using 3D volume, either from X-ray CT, FIB-SEM or numerically generated) and P2D-Pybamm.
My initial aim is to validate the microstructure scale model using a numerically generated sphere-based geometry for which the P2D assumptions are correct. Then, to move on real microstructures and calculate the difference between the two models, that would be attributed to the macroscale model assumptions being no more accurate (non-ideal particle shape + large in-plane heterogeneity).

To provide a fair comparison, I need to provide very accurate microstructure parameters that match the 3D geometry used in my microscale model. My issue lies on the specific surface area: my mesh does not follow the standard 3/r*(1-porosity) law, and thus I would like to enter diameter and specific surface area independently. I saw it is discussed in https://github.com/pybamm-team/PyBaMM/discussions/1769 but it seems the question is not closed. Could you provide information on how to set a fixed specific surface area ?

Best,
Francois.

**Comment by dion-w** [comment]
Yes, that may be confusing in the issue you mentioned. Actually it is possible in the current release to separate the particle size and the effective surface area. It is treated by the parameter "utilisation". Utilisation is actually the effective surface area divided by 3 eps/R (explained in detail in the issue mentioned by you). This ratio is needed to scale the current between the electrolyte scale and the particle scale. We're basically treating non spherical particle all as spherical particles, but we account for the "non-sphericalness" in the utilisation.

**Reply by fussegli** [reply]
Thanks for the answer, the parameter utilization makes sense to me, although I struggle to use it correctly:

With 

```
options
{'intercalation kinetics': 'symmetric Butler-Volmer',
 'thermal': 'isothermal',
 'calculate heat source for isothermal models': 'false',
 'particle mechanics': 'none',
 'stress-induced diffusion': 'false',
 'loss of active material': 'none',
 'lithium plating': 'none',
 'SEI': 'none',
 'SEI film resistance': 'none',
 'SEI porosity change': 'false',
 'operating mode': 'current',
 'working electrode': 'both',
 'particle': 'Fickian diffusion',
 'particle shape': 'spherical',
 'particle size': 'single'}
```

and
`model = pybamm.lithium_ion.DFN(options=options)
`


Then, there is no "utilisation" parameter

```
parameter_values = pybamm.ParameterValues("Chen2020")
parameter_values.search("utilisation")
No results for search using 'utilisation'. Best matches are ['Current function [A]', 'Initial inner SEI thickness [m]', 'Negative electrode electrons in reaction']
```

In https://github.com/pybamm-team/PyBaMM/discussions/1769 it is said to use submodel "particle shape" : "user"
So i tried, but then i get an error for
```
model = pybamm.lithium_ion.DFN(options=options)
OptionError: 
'user' is not recognized in option 'particle shape'. Possible values are ['spherical', 'no particles']
```

So, then i tried with 'particle shape': 'no particles', but get the following error:

```
model = pybamm.lithium_ion.DFN(options=options)
AttributeError: 'DomainLithiumIonParameters' object has no attribute 'a_typ'
```

Obviously, I am doing something wrong, could you indicate me where is my error ?
Best regards,
Francois
